tests/unit_tests/test_array_contains.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def array_contains_process_code():
+    return load_process_code("array_contains")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({'data': [1,2,3], 'value': 2}, True),
+        ({'data': ["A","B","C"], 'value': "b"}, False),
+        ({'data': [1,2,3], 'value': "2"}, False),
+        ({'data': [1,2,None], 'value': None}, False),
```

Comment:
This is not in line with [Example 4](https://docs.openeo.cloud/processes/#array_contains), however it seems that either example 4 or operation definition is wrong (last bullet).

Reply:
I went with the assumption that the definition is correct